Source/EasyNetQ/Interception/CompositeProduceConsumerInterceptor.cs

```diff
@@ -12,21 +12,7 @@ public CompositeProduceConsumerInterceptor(IEnumerable<IProduceConsumeIntercepto
         this.interceptors = interceptors.ToArray();
     }
 
-    public ProducedMessage OnProduce(in ProducedMessage message)
```

Comment:
File can be removed now.

Reply:
Let's leave it, just for those, who uses IServiceRegister and wants to register multiple interceptor in this way 😉

Reply:
ok